docs/cdk-erigon/deploy-testnet.md

```diff
@@ -1,208 +1,1007 @@
+  # If you require assistance deploying and configuring a robust DAC for your Validium chain, Gateway.fm offers support and managed solutions. ([https://gateway.fm](https://gateway.fm))
+
+[Aggregator] # If cdk-node also handles aggregation with a prover
+  Enabled = true # or false if only sequencing
+  ProverURL = "http://localhost:50071" # URL of your zkevm-prover (mock or real)
+  # Aggregation parameters
+
+# ... other sections as per cdk-node documentation ...
 ```
+*   *Examine the `kurtosis-cdk` files that configure and launch the `cdk-node` equivalent (e.g., `cdk_sequencer_node.star` or similar) to understand its specific config file structure and parameters, especially for DAC integration.*
 
-## Inspecting and Interacting with the Network
-
-### View Network Layout
-
-The following commands work on both macOS and Linux:
+## Phase 3: Building and Running Components
 
+### 1. Build `cdk-erigon`
 ```bash
-kurtosis enclave inspect cdk
+git clone https://github.com/0xPolygonHermez/cdk-erigon.git
+cd cdk-erigon
+# Ensure Go version is correct, and C++ build tools (g++, libgmp-dev, libomp-dev) are installed
+make build-libs # If on x86 Linux for optimal performance. For other architectures (e.g., macOS ARM), this step might be skipped or have alternatives; consult cdk-erigon documentation if issues arise.
+make cdk-erigon
+# The binary will be in ./build/bin/cdk-erigon
 ```
 
-This command shows all running services and their ports.
-
-### Set Environment Variables for Tools
-
+### 2. Build `cdk-node`
+The `cdk-node` repository is typically `0xPolygon/cdk`.
 ```bash
-# Get the RPC endpoint for an Erigon node
-export ETH_RPC_URL="$(kurtosis port print cdk cdk-erigon-node-001 rpc)"
-
-# Verify connection by checking block number
-cast block-number
+git clone https://github.com/0xPolygon/cdk.git
+cd cdk/cdk-node # Or the relevant subdirectory for the node
+# Follow build instructions, likely using 'go build' or a Makefile
+make build # Or similar, check its README
+# The binary will be in a 'dist' or 'build' folder
 ```
 
-### Find Important Service Endpoints
-
-The following commands work on both macOS and Linux:
+### 3. Running `cdk-erigon`
+Ensure your L2 configuration directory (e.g., `my-cdk-chain-config`) with `dynamic-{yournetwork}.yaml` and the genesis JSON files is accessible.
 
 ```bash
-# L1 Node RPC endpoint
-kurtosis port print cdk cdk-erigon-node-001 rpc
-
-# Sequencer RPC endpoint
-kurtosis port print cdk cdk-erigon-sequencer-001 rpc
-
-# Sequencer Data Streamer endpoint
-kurtosis port print cdk cdk-erigon-sequencer-001 data-streamer
+# Set this if cdk-erigon will be the sequencer
+export CDK_ERIGON_SEQUENCER=1
+
+# Path to your cdk-erigon binary and config file
+/path/to/cdk-erigon/build/bin/cdk-erigon \
+  --config=/path/to/my-cdk-chain-config/dynamic-{yournetwork}.yaml \
+  --zkevm.genesis-config-path=/path/to/my-cdk-chain-config/ # Directory of allocs, chainspec etc.
```

Comment:
We can drop this flag and just rely on those files being relative to the yaml file